compressor/tests/test_offline.py

```diff
@@ -364,6 +376,21 @@ class OfflineCompressStaticTemplateTagTestCase(OfflineTestCaseMixin, TestCase):
     templates_dir = 'test_static_templatetag'
     expected_hash = 'dfa2bb387fa8'
 
+    def _test_offline(self, engine):
+        count, result = CompressCommand().compress(
+            log=self.log, verbosity=self.verbosity, engine=engine
+        )
+        self.assertEqual(1, count)
+        self.assertEqual([self._render_script(self.expected_hash)], result)
+        self.assertEqual(
+            self._render_template(engine), self._render_result(result))
+
+        # Changing settings.STATIC_URL doesn't affect the next
+        # offline "decompression"
+        with self.settings(STATIC_URL='/another/static/url/'):
+            self.assertEqual(
+                self._render_template(engine), self._render_result(result))
```

Comment:
Sure, I'll do a separate test class.

By "offline decompression" I mean:

* offline manifest key determination
* extraction & processing of the offline manifest content into the HTML output